extension/options.html

```diff
@@ -80,6 +86,13 @@ <h1>Open in Browser</h1>
 handler for the given document.</p>
 
 <h2>Preferences</h2>
+<div class="row">
+  <button id="export" type="button">Export</button>
+  <button id="import" type="button">Import</button>
```

Comment:
These buttons do not just export/import "preferences", but all settings. So please move it to a new section after the last row.